.github/workflows/testing.yml

```diff
@@ -238,7 +233,7 @@ jobs:
       - name: Install deps
         shell: bash
         run: |
-          ./.github/scripts/install_deps.sh ${{ matrix.os }}
+          ./.github/scripts/install_deps.sh ubuntu
```

Comment:
Please add a comment so it's clear why this is needed and that it isn't a copy+paste mistake.

Reply:
What would be? I'm confused.

Reply:
This is the build for windows-mingw-build, running on ubuntu-22.04. If `matrix.os` was already expanding to `ubuntu-22.04`, I'm not sure why this was changed. (Note that the install_deps.sh script checks `case "${1%-*}"`, trimming off a hyphen and everything after it, because we normally pass the `os` value to it.)

Reply:
It wasn't, actually; since this only runs on `ubuntu-22.04`, we *removed* the `matrix:`, so this actually expanded *to nothing*, causing `set -u` to fire inside of `install_deps.sh` when I enabled that.

It's amusing how the only dependency of that list that we were using for cross-compiling, Bison, was already installed by default—so we didn't notice :D